docs/source/subprojects.rst

```diff
@@ -1,9 +1,9 @@
 .. _subprojects:
 
 
-========
-Projects
-========
+================
+Jupyter projects
```

Comment:
This looks like the same content as doc-proj-categories, with some more explanation about each. Too much duplication!